Add Casper OSS List (#721)
casper-network/docs: source/docs/casper/dapp-dev-guide/build-on-casper/casper-open-source-software.md

# Ecosystem Open-Source Software

> This page lists down Casper's thriving ecosystem of Open-Source Softwares that can be utilized by the developers and the community to develop solutions and projects built upon them.

Reviewer: ```suggestion
This page lists Open-Source Software available within the Casper ecosystem. Developers and the community may use the listed software as a basis for new projects and solutions.
```

Author: Thanks @ACStoneCL , updated as suggested

Name | Description | Author | Language | License | Last Update Date | Type
-- | -- | -- | -- | -- | -- | --
[Basic Yield Farm](https://github.com/Rengo-Labs/CasperLabs-StakingRewards) | Staking Rewards | David Tai | Rust | Apache-2.0 license | 2022-06-24 | Staking
[Blockcerts on Casper](https://github.com/amazanzan/casper-cert-issuer) | Issues Blockcerts using the Bitcoin, Ethereum, or Casper blockchain | amazanzan | Python | MIT license | 2022-07-22 | Tokens

> [BlockMatcher](https://blockmatcher.ledgerleap.com/)[Frontend](https://github.com/ledgerleapllc/blockmatcher-frontend) [Backend](https://github.com/ledgerleapllc/blockmatcher-backend) | Private Trade (OTC) Platform for Compliant Brokers.This system allows the creation of OTC token deals, matching buyers and sellers together. The admin, or broker, can then clear them in batches. | LedgerLeap | JavaScript-PHP | MIT license | 2022-01-15 | Exchange

Reviewer: ```suggestion
[BlockMatcher](https://blockmatcher.ledgerleap.com/)[Frontend](https://github.com/ledgerleapllc/blockmatcher-frontend) [Backend](https://github.com/ledgerleapllc/blockmatcher-backend) | Private Trade (OTC) Platform for Compliant Brokers. This system allows the creation of OTC token deals, matching buyers and sellers together. The admin, or broker, can then clear them in batches. | LedgerLeap | JavaScript-PHP | MIT license | 2022-01-15 | Exchange
```

Author: Thanks @ACStoneCL , updated in the latest commit

[Casper Dart SDK](https://github.com/cdolaz/casper_dart_sdk) | Casper Dart SDK is for interacting with the Casper Blockchain using RPC. | Temiltas | Dart | Apache License 2.0 | 2022-06-26 | Client SDK
[Casper ERC20](https://github.com/FriendlyMarket/casper-erc20) | Implementation of the ERC20 standard on the Casper Network | Friendly Market | Rust | Apache-2.0 license | 2022-08-10 | Tokens
[Casper Go SDK](https://github.com/casper-ecosystem/casper-golang-sdk) | Casper Go SDK | Yaroslav Panasenko | Go | Apache-2.0 license | 2022-01-31 | Client SDK
[Casper Hello World Contract](https://github.com/casper-ecosystem/hello-world) | This example demonstrates the session code that accepts a message string and stores it in the calling account under the special_value NamedKey | CasperLabs | Rust | NA | 2022-07-13 | Example Contrats
[Casper JavaScript SDK](https://github.com/casper-ecosystem/casper-js-sdk) | Casper Client JavaScript SDK | CasperLabs | TypeScript-JavaScript | Apache License 2.0 | 2022-09-13 | Client SDK

> [Casper Key-Value Store Contract](https://github.com/casper-ecosystem/kv-storage-contract) | This example demonstrates how Key-Value contract works | CasperLabs | Rust | NA | 2021-10-05 | Example Contrats

Reviewer: Are we still trying to remove the Key-Value Storage contract from public facing docs?

Reviewer 2: @devendran-m I can confirm this, we removed it from the docs, so it shouldn't be in this list.

Author: Thanks @ipopescu , I've removed it and I will push the commit soon

[Clearest Stake](https://github.com/ledgerleapllc/cleareststake-frontend) | Staking Asset Manager for Node operators and groups | LedgerLeap | PHP | Apache-2.0 license | 2022-04-02 | DeFi
[DAO Contracts](https://github.com/make-software/dao-contracts) | Smart Contracts for the MVPR On-Chain Governance System on Casper | MAKE Software | Rust | Apache License 2.0 | 2022-07-17 | DAO
[DHF PAY](https://github.com/DHFinance/dhf-pay-front) | The crypto currency payment gateway on the CSPR blockchain | DHFinance | TypeScript | Apache-2.0 license | 2022-08-28 | Payments
[DLN DAO](https://github.com/dlndao/start/) | DLN DAO | AlphaFin | TypeScript | MIT license | 2022-09-10 | DAO
[DotOracle](https://github.com/dotoracle/bridge-ui) | Realtime decentralized Oracle and Cross-chain liquidity network for EVM, non EVM Casper and L2 blockchains. | DotOracle Network | TypeScript | NA | 2022-09-26 | Bridge
[Dragon’s Lair Style Staking](https://github.com/Rengo-Labs/CasperLabs-Dragonlair) | Lair Style Staking | David Tai | Rust | Apache-2.0 license | 2022-06-24 | Staking

> [ERC1337 Subscription Billing](https://github.com/Rengo-Labs/CasperLabs-EIP1337) | ERC1337 subscription billing | David Tai | Rust | MIT license | 2022-04-07 | Tokens

Reviewer: For the ERC-likes, we may want to phrase it slightly different. Maybe something like "Casper Variant of ERC-1337".

@ipopescu Do you know if we have a standard terminology for these?

Reviewer 2: Since this is not our repo, we cannot rename the repo, but we can easily rename the link. Let's take out ERCxxx like this.
`[Subscription Billing]`

Author: Thanks @ipopescu , done and I will push the commit soon

> [ERC20](https://github.com/casper-ecosystem/erc20) | Implementation of ERC20 token for the CasperLabs platform | CasperLabs | Rust | Apache License 2.0 | 2022-09-01 | Tokens

Reviewer: ```suggestion
[Casper Fungible Token](https://github.com/casper-ecosystem/erc20) | Implementation of fungible tokens for the CasperLabs platform | CasperLabs | Rust | Apache License 2.0 | 2022-09-01 | Tokens
```

Author: Thanks @ACStoneCL , addressed in the updated commit.
